### PR TITLE
Added card component and configured it to accept necessary props

### PR DESCRIPTION
**Pull request by iannho**
Worked on the card component Sean initially set up and included all the required styling from figma

### CODE REVIEW

File: pages/home/_cards.js [hunk 1]
```diff
@@ -13,25 +19,38 @@ const SpeakerContent = styled.div`
...
         </Text>
       </SpeakerContent>
     </SpeakerCard>
   )
 }
 
+Cards.propTypes = {
+  imageSource: PropTypes.string,
+  name: PropTypes.string.isRequired,
+  children: PropTypes.string.isRequired
```
Comment (seanho96) [inline pages/home/_cards.js:53]: PropTypes of `children` should be `node` if not mistaken.

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (seanho96) [inline .vs/VSWorkspaceState.json:1]: I think these `.vs` files should be kept in the .gitignore

Comment (seanho96) [inline .vs/VSWorkspaceState.json:1]: I think these `.vs` files should be kept in the .gitignore

Comment (seanho96) [inline .vs/VSWorkspaceState.json:1]: I think these `.vs` files should be kept in the .gitignore. Same with the 2 files below.

File: pages/home/_cards.js [hunk 3]
```diff
@@ -13,25 +19,38 @@ const SpeakerContent = styled.div`
...
+  margin-top: -51px;
+  z-index: 1;
+`
+const StyledImage = styled(Image)`
+  border-radius: 2.5rem;
 `
 
-const Cards = () => {
+const Cards = (props) => {
+  const { imageSource, name, children } = props
```
Comment (seanho96) [inline pages/home/_cards.js:34]:
```suggestion
  const { src, name, children } = props
```

Comment (seanho96) [reply pages/home/_cards.js:34]: We should try to keep the props naming consistent to what it gets passed to.

File: pages/home/_cards.js (hunk 1 above)
Comment (iannho) [reply pages/home/_cards.js:53]: Ah, I just googled this. Didn't face any errors in those SO posts but will make the necessary changes to make it consistent.

File: pages/home/_cards.js (hunk 3 above)
Comment (iannho) [reply pages/home/_cards.js:34]: The reason why I didn't use the same name was that I wasn't sure if it was good practice to name variables the same as native html parameters. Will change if this is fine.

**Pull request merged**